libs/infinity_emb/infinity_emb/infinity_server.py

```diff
@@ -107,7 +112,7 @@ async def _models():
         return dict(
             data=[
                 dict(
-                    id=engine_args.model_name_or_path,
+                    id=model_name_response_name,
```

Comment:
Also missing this engine_args.served_model_name in `/embeddings` reponse

Reply:
Isn't that done [here](https://github.com/michaelfeil/infinity/blob/main/libs/infinity_emb/infinity_emb/infinity_server.py#L154) already ?

Reply:
Thats right - my bad!